test/src/unit/template.test.js

```diff
@@ -10,6 +10,7 @@
 *******************************************************************************/
 const chai = require('chai');
 
+global.codewind = { RUNNING_IN_K8S: false };
```

Comment:
why's this needed? and shouldn't we unset it after the test (or don't use a global) to decouple tests?

Reply:
Without this, there's an undefined error when loading the `Templates` class, which is due to https://github.com/eclipse/codewind/blob/master/src/pfe/portal/modules/utils/sharedFunctions.js#L19 trying the read the global

I'm not sure how the existing tests were able to run before, they too fail without this.

Setting `RUNNING_IN_K8S` to false explicitly is not needed tho, and I changed that.